scripts/rageshake.go

```diff
@@ -110,6 +127,20 @@ func main() {
 	// Make sure bugs directory exists
 	_ = os.Mkdir("bugs", os.ModePerm)
 
+	// serve files under "bugs"
+	fs := http.FileServer(http.Dir("bugs"))
+	fs = http.StripPrefix("/api/listing/", fs)
+
+	// set auth if env vars exist
+	usr := os.Getenv("BUGS_USER")
+	pass := os.Getenv("BUGS_PASS")
+	if usr == "" || pass == "" {
+		fmt.Println("BUGS_USER and BUGS_PASS env vars not found. No authentication is running for /api/listing")
+	} else {
+		fs = basicAuth(fs, usr, pass, "Enter username and password")
+	}
+	http.Handle("/api/listing/", fs)
```

Comment:
This is not the link that will be present for our deployment. The Apache is configured to route `riot.im/bugreports/*` to `/api/*` so the individual report is linked as `https://riot.im/bugreports/listing/2017-02-07/205222`. I can rename `listing` to `reports` if you want to be redundant.